server/player.lua

```diff
@@ -447,6 +447,17 @@ function QBCore.Player.CreatePlayer(PlayerData, Offline)
         return false
     end
 
+    function self.Functions.updateItemMetaData(item, slot, info)
```

Comment:
```suggestion
    function self.Functions.UpdateItemMetaData(item, slot, info)
```
Please follow the "format standards" in qb-core. Considering all other functions start with capital letters, this should also start with an capital letter :))